src/main/java/org/sagebionetworks/web/client/widget/entity/download/Uploader.java

```diff
@@ -564,11 +570,8 @@ public void setCurrentExternalUploadUrl(String currentExternalUploadUrl) {
   }
 
   public void checkFileSize() throws IllegalArgumentException {
-    JavaScriptObject blob = synapseJsniUtils.getFileBlob(
-      this.currIndex,
-      fileList
-    );
-    long fileSize = (long) synapseJsniUtils.getFileSize(blob);
+    File blob = fileList.item(this.currIndex);
+    long fileSize = blob.size;
```

Comment:
In many cases where it was more simple, I just replaced calls on utils methods with direct calls to the new JsInterop methods